app/api/server/v1/misc.js

```diff
@@ -271,8 +272,10 @@ const methodCall = () => ({
 			const result = Meteor.call(method, ...params);
 			return API.v1.success(mountResult({ id, result }));
 		} catch (error) {
-			Meteor._debug(`Exception while invoking method ${ method }`, error.stack);
-
+			SystemLogger.error('Error starting video call:', error.message);
```

Comment:
```suggestion
			SystemLogger.error(`Exception while invoking method ${ method }`, error.message);
```

This will catch any error from any meteor method call, so the log message should not be related to the video call.